tableauserverclient/server/endpoint/jobs_endpoint.py

```diff
@@ -17,18 +17,7 @@ def baseurl(self):
         return "{0}/sites/{1}/jobs".format(self.parent_srv.baseurl, self.parent_srv.site_id)
 
     @api(version="2.6")
-    def get(
-        self, job_id: Optional[str] = None, req_options: Optional[RequestOptionsBase] = None
-    ) -> Tuple[List[BackgroundJobItem], PaginationItem]:
-        # Backwards Compatibility fix until we rev the major version
-        if job_id is not None and isinstance(job_id, str):
```

Comment:
why removing this now?

Reply:
I saw that it was deprecated and removed it. Though, you are right it is outside of the scope for this PR. Reverting that change.